features/aanschrijfwijze.feature

```diff
@@ -366,41 +392,41 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 999992934                       |
       | fields              | naam.aanschrijfwijze            |
```

Comment:
De fields moet hier naam.aanschrijfwijze.naam zijn denk ik. Anders zit de aanspreekvorm ook in de response